pkg/devicerepository/config.go

```diff
@@ -19,30 +19,45 @@ import (
 
 	"go.thethings.network/lorawan-stack/v3/pkg/config"
 	"go.thethings.network/lorawan-stack/v3/pkg/devicerepository/store"
+	"go.thethings.network/lorawan-stack/v3/pkg/devicerepository/store/bleve"
+	"go.thethings.network/lorawan-stack/v3/pkg/errors"
 )
 
 // Config represents the DeviceRepository configuration.
 type Config struct {
 	Store StoreConfig `name:"store"`
 
-	ConfigSource string                `name:"config-source" description:"Source of the Device Repository (static, directory, url, blob)"`
-	Static       map[string][]byte     `name:"-"`
-	Directory    string                `name:"directory" description:"OS filesystem directory, which contains Device Repository package"`
-	URL          string                `name:"url" description:"URL, which contains Device Repository package"`
-	Blob         config.BlobPathConfig `name:"blob"`
+	ConfigSource string `name:"config-source" description:"Source of the Device Repository (directory)"`
+	Directory    string `name:"directory" description:"OS filesystem directory, which contains Device Repository package"`
```

Comment:
1. As I commented in the other PR, I don't think we're breaking compatibility if we make changes, as this wasn't ever used or documented. Therefore I think we can rename this to `Source` (because it's the device repository source, not the config source).
2. Maybe "Device Repository package" should be "Device Repository source"